setup.py

```diff
@@ -514,6 +515,22 @@ def configuration(parent_package='', top_path=None):
 
 
 def setup_package():
+    # In maintenance branch, change np_maxversion to N+3 if numpy is at N
+    # Update here and in scipy/__init__.py
+    # Rationale: SciPy builds without deprecation warnings with N; deprecations
+    #            in N+1 will turn into errors in N+3
+    # For Python versions, if releases is (e.g.) <=3.9.x, set bound to 3.10
+    np_minversion = '1.16.5'
```

Comment:
Should this be a value from oldes-supported-numpy?

Reply:
no, see comment above.